libraries/core-react/src/components/Menu/MenuList.tsx

```diff
@@ -41,28 +34,41 @@ type MenuChild = ReactElement<MenuItemProps> & ReactElement<MenuSectionProps>
 
 type Direction = 'down' | 'up'
 
+function isIndexable(item: MenuChild) {
+  if (isValidElement(item) && !item.props.disabled && item.type === MenuItem)
+    return true
+  return false
+}
+
 export const MenuList = forwardRef<HTMLUListElement, MenuListProps>(
   function MenuList({ children, focus, ...rest }, ref) {
     const { focusedIndex, setFocusedIndex } = useMenu()
 
-    const pickedChildren = isFragment(children)
-      ? (children as MenuChild).props.children
-      : children
-
-    const updatedChildren: Array<MenuChild> = ReactChildren.map(
-      pickedChildren,
-      (child: ReactNode, index: number) =>
-        cloneElement(child as MenuChild, { index }),
+    let index = -1
+    const focusableIndexs: number[] = useMemo<number[]>(() => [], [])
```

Comment:
Just curious, whats the benefit of putting memonizing this constant when is used from a memonized function?

Reply:
The linter told me to do it:
`The 'focusableIndexs' array makes the dependencies of useMemo Hook (at line 70) change on every render. To fix this, wrap the initialization of 'focusableIndexs' in its own useMemo() Hook.`

Reply:
aha, probably because its declared in the function, which means that it would be re-declared on every render 🤔